location/src/main/java/com/yas/location/repository/CountryRepository.java

```diff
@@ -18,8 +18,19 @@ WHEN count(1)> 0 THEN TRUE
                    ELSE FALSE
                 END
          FROM Country c
-         WHERE c.name = ?1
+         WHERE LOWER(c.name) = LOWER(?1)
          AND c.id != ?2
```

Comment:
Avoid using the function in the WHERE clause. It's not good for performance.
Ref: https://www.mssqltips.com/sqlservertutorial/3204/avoid-using-functions-in-where-clause/